uni/lib/utils/drawer_items.dart

```diff
@@ -7,7 +7,8 @@ enum DrawerItem {
   navLocations('locais', faculties: {'feup'}),
   navRestaurants('restaurantes'),
   navCalendar('calendario'),
-  navLibrary('biblioteca', faculties: {'feup'}),
+  navLibraryOccupation('biblioteca', faculties: {'feup'}),
+  navLibraryReservations('reservas', faculties: {'feup'}),
```

Comment:
Why does this need to be here? Isn't this on the same page?

Reply:
I created a different route for each tab. I am using this for each card's `onClick`